src/cli/commands/build.ts

```diff
@@ -28,6 +28,7 @@ export default function registerCommand(program: Command) {
     .option("-I, --api-build-command <command>", "the command used to build your api", DEFAULT_CONFIG.apiBuildCommand)
     .option("--auto", "automatically detect how to build your app and api", false)
     .action(async (positionalArg: string | undefined, _options: SWACLIConfig, command: Command) => {
+      positionalArg = positionalArg?.trim();
```

Comment:
@sinedied Thanks! for raising the fix here. 

Will it be possible to add few tests for these changes ?

Reply:
Sure!